ci/deb/build-dep.sh

```diff
@@ -82,8 +82,8 @@ echo "INSTALL: rapidjson" \
 && mv /opt/rapidjson-1.1.0/include/rapidjson/ /usr/local/include
 
 echo "INSTALL: libmicrohttpd" \
-&& curl -L https://ftp.gnu.org/gnu/libmicrohttpd/libmicrohttpd-0.9.73.tar.gz | tar xzC /opt/ \
-&& cd /opt/libmicrohttpd-0.9.73  \
+&& curl -L https://ftp.gnu.org/gnu/libmicrohttpd/libmicrohttpd-0.9.75.tar.gz | tar xzC /opt/ \
+&& cd /opt/libmicrohttpd-0.9.75  \
```

Comment:
Given this PR "breaks the barrier" of release 0.9.75, I'd suggest to use the newest version at this moment, which looking to https://ftp.gnu.org/gnu/libmicrohttpd/ happens to be 0.9.76.

Could you test to change 0.9.75 -> 0.9.76 in all places to check if test are still passing with that one, please?

Reply:
> Could you test to change 0.9.75 -> 0.9.76 in all places to check if test are still passing with that one, please?

I have tested with 0.9.76 all test cases are passing with that.